test/parallel/test-repl-preprocess-top-level-await.js

```diff
@@ -116,6 +117,9 @@ const testCases = [
     '(async () => { for (let i in {x:1}) { await 1 } })()'],
   [ 'for (const i in {x:1}) { await 1 }',
     '(async () => { for (const i in {x:1}) { await 1 } })()'],
+  [ 'var x = await foo(); async function foo() { return Promise.resolve(1);}',
+    'var x; var foo; (async () => {void (x = await foo()); this.foo = foo; ' +
```

Comment:
```suggestion
    'var x; var foo; (async () => { void (x = await foo()); this.foo = foo; ' +
```
This should fix the test failure